irohad/ametsuchi/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 add_library(ametsuchi
-    impl/flat_file/flat_file.cpp
+        impl/flat_file/flat_file.cpp
```

Comment:
Indent is not consistent :( Please check your IDE settings.

Reply:
Hmm, very strange issues with indentation, when I push I see perfectly fine picture, but after that such things happen... I'll check it